logger/options.go

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
LoggerConfig has many default values so that ```WithZap(), WithLevelInfo(), WithFormatText(), WithAppenderConsole()``` could be deleted?

Reply:
I will replace these methods with some more generic methods instead. I did check all the possible values of each type, and they seems to very limited as I listed according to the implementation.

Reply:
But I will still keep `WithZap()` and `WithLogurus()` as they are currently the two drivers we support.